### DISCUSSION TITLE
licensing question

### DISCUSSION THREAD

**Discussion by studioph** [Eloston/ungoogled-chromium#1502] [General]
I'd like to make a docker image of ug-chromium similar to [jlesage/docker-firefox](https://github.com/jlesage/docker-firefox) where the gui is accessed via a browser. I was looking over the license in the repo and had a few questions (I'm new a very much a noob when it comes to open source licensing so I want to make sure I'm doing this correctly).

The 2nd clause states: `Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.` Is having a link to the ug-chroimium license in my readme sufficient, or would i need to include the license file within the docker image?

I'm a little confused by the 3rd clause: `Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.` Am I allowed to call my docker image something like "Ungoogled Chromium Docker", or do I need to call it something else?

I would greatly appreciate any clarification or information on this, thanks!

**Comment by Ahrotahn** [comment]
I'm not a legal expert, so this is not legal advice.  I also cannot speak on behalf of Eloston or the ungoogled-chromium project.  

That being said, my understanding of the license is that a link would not work.  You would need to include the full text of the license.  
[TLDRLegal](https://tldrlegal.com/license/bsd-3-clause-license-(revised)) has a good breakdown for laymen.  

For the third clause, there is an answer [here](https://opensource.stackexchange.com/a/4630) that explains that you should be fine so long as you don't imply that the project is endorsed by ungoogled-chromium or it's contributors.  

Hopefully someone with more expertise on the matter can offer further insight for you!

**Reply by studioph** [reply]
Hey, thanks for the info and sources! This gives me a better understanding of how to move forward.
